### DISCUSSION TITLE
Can you create an option so we can install on pinokio?

### DISCUSSION THREAD

**Discussion by xenstar** [unclecode/crawl4ai#521] [Feature requests]
Hello, can you make a version to easily install it on https://pinokio.computer/?

**Comment by unclecode** [comment]
Very interesting. Actually, it has been on my mind about it. This week we're releasing a number of changes in the new version. I'll be quite busy testing and answering people's questions. I think by next week, I'll definitely go through it and try it out. It's actually a very good way and helpful for people. Thank you for bringing it up.
